builder/tart/step_disk_resize.go

```diff
@@ -13,13 +13,21 @@ import (
 type stepResize struct{}
 
 func (s *stepResize) Run(ctx context.Context, state multistep.StateBag) multistep.StepAction {
+	config := state.Get("config").(*Config)
 	ui := state.Get("ui").(packersdk.Ui)
 
 	_, diskChanged := state.GetOk(statekey.DiskChanged)
 	if !diskChanged {
 		return multistep.ActionContinue
 	}
 
+	// Skip in-guest disk resizing for ASIF disks
```

Comment:
Even taking into account the `if config.DiskFormat == "asif"` check in `step_disk_file_prepare.go`, I still don't quite understand why we need to skip this here too.

This looks like a purely in-guest operation not requiring any knowledge of the disk format on host.